.teamcity/src/main/kotlin/common/JvmVersion.kt

```diff
@@ -18,12 +18,7 @@ package common
 
 enum class JvmVersion(val major: Int) {
     java8(8),
-    java9(9),
```

Comment:
Did you remove those constants because they are not installed any more?

Reply:
Yes, and they're used to generate a start parameter `-Porg.gradle.java.installation.path=XXX`:

https://github.com/gradle/gradle/blob/e9f2c1ee8c3d23d8f6c9b1042f1b2e964791af40/.teamcity/src/main/kotlin/common/Os.kt#L90